recipe/meta.yaml

```diff
@@ -7,19 +7,20 @@ package:
 source:
   # for local testing use a tarball including submodules
   git_url: https://github.com/pytorch/pytorch.git
-  git_tag: v{{ version }}
+#   git_tag: v{{ version }}
   patches:
     - fix_dispatch_apply_auto.patch
 
 build:
   number: 1
-  skip: true  # [win]
+  skip: true  # [not osx]
 
 outputs:
   - name: pytorch
     build:
       string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
-      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
+      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None" and osx_impl != "mps"]
+      string: mps_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None" and osx_impl == "mps"]
```

Comment:
I think there is a similar build number below. You might want to add an other `pytorch-cpu` package for MAC and just pin it to the MPS package.

Reply:
Updated, it's a bit of matrix now... 😰